### PR TITLE
Remove references to legacy subscription plans

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request removes references to legacy subscription plans ([Cloud Flex Accounts](https://help.sumologic.com/docs/manage/manage-subscription/cloud-flex-accounts/)), which are no longer being sold, such as the Professional account type.

Issue: #3245 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/api/troubleshooting.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ API- 403 error occurs when you do not have access to the Search Job API.  For ac
 
 #### Answer
 
-The Search Job APIs is only available within Enterprise level accounts. If you are currently on Sumo Logic Free or Professional accounts, you receive this error message if you try to access these APIs. If you need access to these APIs please reach out to your Sumo Logic Sales Associate at sales@sumologic.com or [contact us](https://www.sumologic.com/contact-us/) for more information on upgrading your existing account.
+The Search Job APIs is only available within Enterprise level accounts. If you are currently on a Sumo Logic Free account, you receive this error message if you try to access these APIs. If you need access to these APIs please reach out to your Sumo Logic Sales Associate at sales@sumologic.com or [contact us](https://www.sumologic.com/contact-us/) for more information on upgrading your existing account.
```
Comment (kimsauce) [inline docs/api/troubleshooting.md:92]:
```suggestion
The Search Job APIs is only available within Enterprise level accounts. If you are currently on a Sumo Logic Free account, you receive this error message if you try to access these APIs. If you need access to these APIs please reach out to your Sumo Logic Sales Associate at [sales@sumologic.com](mailto:sales@sumologic.com) or [contact us](https://www.sumologic.com/contact-us/) for more information on upgrading your existing account.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix! Was this PR meant to remove Professional account references from all docs? I ran a search and have found more references @jpipkin1 .

**Comment by jpipkin1** [comment]
>minor fix! Was this PR meant to remove Professional account references from all docs? I ran a search and have found more references @jpipkin1 .

Yes, @kimsauce. This PR was meant to remove Professional account references. Could you share with me the references you found?

**Comment by kimsauce** [comment]
> > minor fix! Was this PR meant to remove Professional account references from all docs? I ran a search and have found more references @jpipkin1 .
> 
> Yes, @kimsauce. This PR was meant to remove Professional account references with the exception of those in the [Cloud Flex Accounts](https://help.sumologic.com/docs/manage/manage-subscription/cloud-flex-accounts/) and the [Upgrade Your Cloud Flex Account](https://help.sumologic.com/docs/manage/manage-subscription/upgrade-cloud-flex-account) articles, since those articles will continue to document the legacy account types.
> 
> If you found references in other articles, could you share them with me?

sure thing! I just ran a search for "Professional" (case sensitive) and found some here:
Screenshot 2023-10-23 at 2 52 11 PM

**Comment by jpipkin1** [comment]
Thank you for the list, @kimsauce. Many of the articles contained "Professional Services", but many contained citations for Professional accounts. I'll put this ticket back into draft so I can work on it some more.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**